json-schemas/protocol-rule-set.json

```diff
@@ -32,6 +32,35 @@
             }
           }
         },
+        {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "author": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "of": {
+                  "type": "string"
+                },
+                "to": {
+                  "type": "array",
+                  "minItems": 1,
+                  "items": {
+                    "type": "string",
+                    "enum": [
+                      "write"
+                    ]
+                  }
+                }
```

Comment:
Consider filing a good-first-issue for the community to refactor the repeated definition into its own schema so we can just reference the same schema.

Reply:
Good call. #286